modMcf/test/org/aion/mcf/account/KeystoreTest.java

```diff
@@ -99,6 +96,7 @@ public void keyCreateAndRetrieve2() {
     @Test
     public void testKeyCreate() {
         String password = randomPassword();
+
```

Comment:
My comments on AccountManager about temp file and unit vs. integ test apply here also.  I know you didn't create this test originally, but can you try to figure out a way to clean up all the files created by this test?  I think every time the Keystore.create method gets hit, a file gets created on the file system.

Reply:
sure, ill check where files are created and make sure they are removed after the test functions.